docs/conf.py

```diff
@@ -191,6 +192,8 @@ def current_version() -> str:
 
 smartquotes = False
 
+inverter_all = False
+
```

Comment:
what it does?

Reply:
disables by default inverting all images, so only the ones we explicitly set will do that